tests/output_filters/related_files_test.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 import testify as T
 from osxcollector.output_filters.related_files import RelatedFilesFilter
-from tests.output_filters.run_filter_test import assert_equal_sorted
 from tests.output_filters.run_filter_test import RunFilterTest
+from tests.output_filters.run_filter_test import assert_equal_sorted
```

Comment:
These reorders are the pre-commit hooks. I decided to push them as is vs. mucking with pre-commit.